atomics/T1049/T1049.yaml

```diff
@@ -49,5 +49,5 @@ atomic_tests:
     name: sh
     elevation_required: false
     command: |
-      netstat
+      if [ -x "$(command -v netstat)" ]; then netstat; else echo "netstat is missing from the machine. skipping..."; fi;
```

Comment:
Could we add the check for binary existing as actual prereqs as you do in some of the later files in this commit?

Reply:
point taken